src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentRepository.php

```diff
@@ -34,7 +35,7 @@ public function createListQueryBuilder(): QueryBuilder
         ;
     }
 
-    public function findOneByOrderId($paymentId, $orderId): ?PaymentInterface
+    public function findOneByOrderId(string|int $paymentId, string $orderId): ?PaymentInterface
```

Comment:
```suggestion
    public function findOneByOrderId(string|int $paymentId, string|int $orderId): ?PaymentInterface
```
Pretty sure we'll need to go through all the typehints regarding ids and make them looser, maybe even mixed.